### PR TITLE
Fix RDMA on SLES15SP1.

### PR DESCRIPTION
**Pull request by LiliDeng**
@juhlee-microsoft  Any other sles sku support RDMA, need run regression testing?
Test result -

```
[LISAv2 Test Results Summary]
Test Run On           : 05/25/2020 14:00:31
ARM Image Under Test  : SUSE : sles-15-sp1 : gen1 : latest
Initial Kernel Version: 4.12.14-8.22-azure
Final Kernel Version  : 4.12.14-8.22-azure
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:21

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 INFINIBAND           INFINIBAND-OPEN-MPI-2VM                                                           PASS                15.43 
	InfiniBand-Verification-1-FirstBoot : eth0 IP : PASS 
	InfiniBand-Verification-1-FirstBoot : IBV_PINGPONG : PASS 
	InfiniBand-Verification-1-FirstBoot :IMB PingPong Intranode : PASS 
	InfiniBand-Verification-1-FirstBoot : IMB-MPI1 : PASS 
	InfiniBand-Verification-1-FirstBoot : IMB-P2P : PASS 
	InfiniBand-Verification-1-FirstBoot : IMB-NBC : PASS 
	InfiniBand-Verification-2-Reboot : eth0 IP : PASS 
	InfiniBand-Verification-2-Reboot : IBV_PINGPONG : PASS 
	InfiniBand-Verification-2-Reboot :IMB PingPong Intranode : PASS 
	InfiniBand-Verification-2-Reboot : IMB-MPI1 : PASS 
	InfiniBand-Verification-2-Reboot : IMB-P2P : PASS 
	InfiniBand-Verification-2-Reboot : IMB-NBC : PASS 

```

### CODE REVIEW

File: Testscripts/Linux/SetupRDMA.sh [hunk 1]
```diff
@@ -187,7 +187,8 @@ function Main() {
 		suse*|sles*)
 			# install required packages
 			LogMsg "Starting RDMA setup for SUSE"
-			req_pkg="bzip expect glibc-32bit glibc-devel libgcc_s1 libgcc_s1-32bit libpciaccess-devel gcc-c++ gcc-fortran rdma-core libibverbs-devel librdmacm1 libibverbs-utils bison flex numactl"
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/SetupRDMA.sh:190]: You removed one 'numactl' recently added. Please sync your branch before development starts

**Comment by juhlee-microsoft** [comment]
SLES 12 SP5 + MVAPICH and SLES 15 SP1 + Intel MPI

**Comment by LiliDeng** [comment]
SLES 15 SP1 + INFINIBAND-INTEL-MPI-2VM 
```
[LISAv2 Test Results Summary]
Test Run On           : 05/26/2020 00:59:44
ARM Image Under Test  : SUSE : sles-15-sp1 : gen1 : latest
Initial Kernel Version: 4.12.14-8.22-azure
Final Kernel Version  : 4.12.14-8.22-azure
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:2:44

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 INFINIBAND           INFINIBAND-INTEL-MPI-2VM                                                          PASS               153.91 
	InfiniBand-Verification-1-FirstBoot : eth0 IP : PASS 
	InfiniBand-Verification-1-FirstBoot : IBV_PINGPONG : PASS 
	InfiniBand-Verification-1-FirstBoot :IMB PingPong Intranode : PASS 
	InfiniBand-Verification-1-FirstBoot : IMB-MPI1 : PASS 
	InfiniBand-Verification-1-FirstBoot : IMB-IO : SKIPPED 
	InfiniBand-Verification-1-FirstBoot : IMB-RMA : PASS 
	InfiniBand-Verification-1-FirstBoot : IMB-NBC : PASS 
	InfiniBand-Verification-2-Reboot : eth0 IP : PASS 
	InfiniBand-Verification-2-Reboot : IBV_PINGPONG : PASS 
	InfiniBand-Verification-2-Reboot :IMB PingPong Intranode : PASS 
	InfiniBand-Verification-2-Reboot : IMB-MPI1 : PASS 
	InfiniBand-Verification-2-Reboot : IMB-IO : SKIPPED 
	InfiniBand-Verification-2-Reboot : IMB-RMA : PASS 
	InfiniBand-Verification-2-Reboot : IMB-NBC : PASS 
```

**Comment by LiliDeng** [comment]
@juhlee-microsoft  SLES12 SP5 also abort in master branch, so I can't provide results for SLES12 SP5, I will file a bug for it.

**Review by juhlee-microsoft — APPROVED** [review]

**Pull request merged**